source/analysis/AMNormalizationAB.h

```diff
@@ -0,0 +1,100 @@
+#ifndef AMNORMALIZATIONAB_H
+#define AMNORMALIZATIONAB_H
+
+#include "analysis/AMStandardAnalysisBlock.h"
+
+/// This analysis block accepts any number of identically ranked data sources and divides one by the other to normalize one designated as "data".
```

Comment:
Yup.  You can pass in a list of data sources and then choose which two you want to analyze.